lib/oci_utils/impl/oci-network-config-main.py

```diff
@@ -536,14 +530,11 @@ def main():
                 ", ".join(excludes))
 
     if args.auto or args.create_vnic or args.add_private_ip:
-        (ret, out) = vnic_utils.auto_config(quiet=args.quiet, show=args.show,
-                                            sec_ip=args.sec_ip)
+        (ret, out) = vnic_utils.auto_config(args.sec_ip)
     elif args.detach_vnic and shape and shape.startswith("BM"):
-        (ret, out) = vnic_utils.auto_config(quiet=args.quiet, show=args.show,
-                                            sec_ip=args.sec_ip)
+        (ret, out) = vnic_utils.auto_config(args.sec_ip)
     elif args.deconfigure:
-        (ret, out) = vnic_utils.auto_deconfig(quiet=args.quiet, show=args.show,
-                                              sec_ip=args.sec_ip)
+        (ret, out) = vnic_utils.auto_deconfig(args.sec_ip)
 
     if ret:
         __logger.error("Failed to execute the VNIC configuration script.")
```

Comment:
maybe the opportunity to change __logger to _logger in this module

Reply:
fixed